src/events/index.spec.js

```diff
@@ -239,6 +239,113 @@ describe('Events', function() {
     });
   });
 
+  describe('getEventTypesByClient', function() {
+    context('the clientId is provided', function() {
+      let clientId = faker.random.uuid();
```

Comment:
The creation of `clientId` should happen in the before each -- and then pass it into `buildList` so you get the same clientId in all the fixtures